modules/data.remote/R/NLCD.R

```diff
@@ -62,7 +62,8 @@ download.NLCD <- function(outdir, year = 2011, con = NULL) {
 ##' @param coords  data frame containing elements 'long' and 'lat'. Currently just supports single point extraction.
 ##' @param data_dir  directory where input data is located. Can be NUL if con is specified
 ##' @param con       connection to PEcAn database. Can be NULL if data_dir is specified
-##' 
+##' @param year one of 2001 or 2011
```

Comment:
if I'm reading the code right, seems like this could be other years if data_dir is provided

Reply:
Good catch! Fixed in ea8a3fdf4